### PR TITLE
Status bar adjustments

### PR DESCRIPTION
**Pull request by victor-dantas**
1. Change Status Bar - webapp
2. Change BhModal Structure - BH
3. Add fade to BhModal - BH
4. Normalize size icons - BH

### CODE REVIEW

File: bh/documentation/src/modal/examples/ModalExample.vue [hunk 1]
```diff
@@ -1,19 +1,18 @@
 <template>
   <div class="bh-grid">
     <div class="bh-grid__item">
-      <div class="bh-grid__item">
+      <div>
```
Comment (douglasppaz) [inline bh/documentation/src/modal/examples/ModalExample.vue]: You can remove this div?!

File: bh/src/assets/icons/sentence.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15.974 15.973"><defs><style>.a{fill:#5d5d5d;}</style></defs><path class="a" d="M7.956,6.3a.624.624,0,1,1-.624.624A.624.624,0,0,1,7.956,6.3Zm2.184.624a.624.624,0,1,0,.624-.624A.624.624,0,0,0,10.14,6.926Zm-5.616,0A.624.624,0,1,0,5.148,6.3.624.624,0,0,0,4.524,6.926Zm10.9,5.852a1.872,1.872,0,0,1-2.647,2.648L9.353,12.008a.623.623,0,0,1-.156-.26L8.451,9.292a.624.624,0,0,1,.764-.783l2.518.7a.624.624,0,0,1,.274.16Zm-5.08-1.543,2.311,2.306.883-.883-2.3-2.293-1.267-.351Zm4.2,2.426-.12-.12-.882.882.12.12a.624.624,0,1,0,.883-.882Zm-6.209-.73c-.115,0-.232.008-.347.008a7.551,7.551,0,0,1-2.871-.555.625.625,0,0,0-.479,0L1.8,13.585l.957-2.268a.624.624,0,0,0-.109-.657,5.29,5.29,0,0,1-1.4-3.551c0-3.231,3.023-5.861,6.739-5.861s6.739,2.629,6.739,5.861A5.136,5.136,0,0,1,14.166,9.4a.624.624,0,0,0,1.119.553,6.373,6.373,0,0,0,.689-2.84C15.974,3.189,12.391,0,7.987,0S0,3.189,0,7.109a6.469,6.469,0,0,0,1.455,4.076L.049,14.515a.624.624,0,0,0,.817.818l4.022-1.7a8.815,8.815,0,0,0,3.1.552c.133,0,.268,0,.4-.009a.624.624,0,1,0-.055-1.247Zm0,0" transform="translate(0 0)"/></svg>
```
Comment (douglasppaz) [inline bh/src/assets/icons/sentence.svg]: remove this class `.a` move to path style...

File: bh/src/assets/icons/warning.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 17.862 16.197"><defs><style>.a{fill:#5d5d5d;}</style></defs><g transform="translate(0 -22.643)"><path class="a" d="M17.505,34.825,11.247,23.986a2.663,2.663,0,0,0-4.625,0L.36,34.825A2.675,2.675,0,0,0,2.673,38.84h12.5A2.689,2.689,0,0,0,17.505,34.825Zm-8.574,1.61a1,1,0,1,1,0-1.993,1,1,0,0,1,0,1.993Zm.908-6.442c-.044.772-.092,1.541-.136,2.313-.022.25-.022.478-.022.724a.75.75,0,0,1-.75.724.734.734,0,0,1-.75-.7c-.066-1.2-.136-2.383-.2-3.585-.022-.316-.044-.636-.07-.952a1.083,1.083,0,0,1,.772-1.088,1,1,0,0,1,1.158.566,1.188,1.188,0,0,1,.092.5C9.909,29,9.861,29.5,9.839,29.994Z" transform="translate(0)"/></g></svg>
```
Comment (douglasppaz) [inline bh/src/assets/icons/warning.svg]: remove this css class `.a`

File: bh/src/assets/scss/buttons.scss [hunk 4]
```diff
@@ -29,6 +29,14 @@
     outline: none;
     box-shadow: .2rem .2rem .5rem rgba(200, 200, 200, .5);
 
+    svg {
```
Comment (douglasppaz) [inline bh/src/assets/scss/buttons.scss]: You can try refer `.bh-icon`

File: bh/src/components/BhModal.vue [hunk 5]
```diff
@@ -1,26 +1,34 @@
 <template>
-  <div
-    v-show="open"
-    ref="modal"
-    class="bh-modal"
-    @click="backgroundClose($event)"
+  <transition
```
Comment (douglasppaz) [inline bh/src/components/BhModal.vue]: put this in one line

File: src/components/repository/RepositoryViewBase/RepositoryViewBase.vue [hunk 6]
```diff
@@ -283,8 +274,16 @@ export default {
       this.training = false;
       await this.updateRepository(false);
     },
-    openTrainModal() {
-      this.trainModalOpen = true;
+    barClicked() {
```
Comment (douglasppaz) [inline src/components/repository/RepositoryViewBase/RepositoryViewBase.vue]: Rename this method to `onStatusBarClick`

File: src/components/repository/TrainModal.vue [hunk 7]
```diff
@@ -101,30 +134,38 @@ export default {
 
 
 .train-modal {
-  padding: .5rem;
+  &__wrapper {
+    background-color: $color-fake-white;
+    padding: 1.5rem;
```
Comment (douglasppaz) [inline src/components/repository/TrainModal.vue]: We don't have global variables to padding?

File: bh/src/assets/scss/buttons.scss (hunk 4 above)
Comment (victor-dantas) [reply bh/src/assets/scss/buttons.scss]: If i refer .bh-icon it does not have the expected behavior.

**Pull request merged**